core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -62,7 +62,9 @@
 -define(DEFAULT_MOBILE_AMQP_CONNECTIONS,
         kz_json:from_list([{<<"default">>, ?DEFAULT_MOBILE_AMQP_CONNECTION}])).
 
--define(CONFIRM_FILE(Call), kz_media_util:get_prompt(<<"ivr-group_confirm">>, Call)).
+-define(CONFIRM_FILE(Call)
+       ,kz_media_util:get_prompt(<<"ivr-group_confirm">>, kapps_call:account_id(Call))
```

Comment:
same here. every former `kz_media_util:get_prompt/2` passing a `Call` object should now call `kz_media_util:get_prompt/3` with `language` and `account_id` of the `Call`. maybe create a `kapps_call:get_prompt/2` (prompt, Call) and havew it call  `kz_media_util:get_prompt/3` ?